node/node.go

```diff
@@ -228,16 +233,21 @@ func New(host p2p.Host, consensusObj *consensus.Consensus, db ethdb.Database) *N
 	if host != nil && consensusObj != nil {
 		// Consensus and associated channel to communicate blocks
 		node.Consensus = consensusObj
-
-		// Init db.
 		database := db
 		if database == nil {
 			database = ethdb.NewMemDatabase()
+			chain, err = node.GenesisBlockSetup(database)
+			isFirstTime = true
+		} else {
+			chain, err = node.InitBlockChainFromDB(db, node.Consensus)
+			isFirstTime = false
+			if err != nil || chain == nil || chain.CurrentBlock().NumberU64() <= 0 {
+				chain, err = node.GenesisBlockSetup(database)
```

Comment:
do you need a log line so that we know what's happening?

Reply:
If the local database is available, we will create the genesis block. it's not an error

Reply:
Yes, I understand.  Please just add an informational log.